web/modules/custom/ai_screening_project_track/src/Computer/WebformSubmissionProjectTrackToolComputer.php

```diff
@@ -25,6 +27,36 @@ public function supports(ProjectTrackToolInterface $tool, EntityInterface $entit
    */
   public function compute(ProjectTrackToolInterface $tool, EntityInterface $entity): void {
     assert($entity instanceof WebformSubmissionInterface);
+
+    $dimensions = $tool->getProjectTrack()->getDimensions();
+    $calculated = [];
+
+    // Loop over form fields values and calculate sums.
+    foreach ($entity->getData() as $value) {
+      try {
+        $fieldValues = FormHelper::getIntegers($value);
+        foreach (array_keys($calculated + $fieldValues) as $key) {
+          $calculated[$key] = ($calculated[$key] ?? 0) + ($fieldValues[$key] ?? 0);
+        }
+      }
+      catch (InvalidValueException $e) {
+        // Form fields without numeric values would end here, but since we
+        // expect this to happen on all non-radio fields we don't take further
+        // actions.
+      }
+    }
+
+    $mappedValues = array_map(function ($key, $value) {
+      return [$key, $value];
+    },
+      $dimensions, $calculated
+    );
```

Comment:
Why do you map to a list of “index“,  “value”, e.g.

``` php
[
 [0, 87],
 [1, 42],
]
```

I have a feeling that just using `$calculated`, i.e. `[0 => 87, 1 => 42]`, may be easier further on.

Reply:
Changed this